Doc/whatsnew/3.12.rst

```diff
@@ -183,6 +183,10 @@ sqlite3
 * Add a :ref:`command-line interface <sqlite3-cli>`.
   (Contributed by Erlend E. Aasland in :gh:`77617`.)
 
+* Add :attr:`~sqlite3.Connection.autocommit` attribute for :pep:`249`-compliant
+  transaction handling.
```

Comment:
```suggestion
* Add the :attr:`~sqlite3.Connection.autocommit` attribute
  to :class:`~sqlite3.Connection`
  and the *autocommit* parameter to :func:`connect`
  to control :pep:`249`-compliant
  :ref:`transaction handling <sqlite3-controlling-transactions-autocommit>`.
```

* Mention that it is added to the factory function as well
* Clarify and link the relevant class the attribute was added to
* Link the new section describing this new transaction handling in more detail

Also, if/when I edit this, I will change it to use past tense (as the other What's New entries generally do), and maybe worth expanding/clarifying a bit more, since the changes are a bit broader than this implies.